src/cnc/tests/test_provisioner.py

```diff
@@ -351,4 +348,135 @@ class AWSProvisionStageTwoServiceServerlessAndTwoResourceDynamoDBTest(
     def test_tf_is_valid(self):
         self.assertEqual(len(self.resources["aws_dynamodb_table"]), 2)
         self.assertEqual(len(self.resources["aws_lambda_function"]), 2)
+
+
+class AWSProvisionStageOneServiceServerlessOneResourceDynamoDBOneDatabaseTest(
+    AWSProvisionStageTestBase
+):
+    fixture_name = "serverless-1-service-1-dynamodb-1-db"
+    env_data_filepath = "environments_serverless_1_service_1_dynamodb_1_db.yml"
+
+    def test_tf_is_valid(self):
+        self.assertEqual(len(self.resources["aws_dynamodb_table"]), 1)
```

Comment:
we should assert that there is no vpc or NAT/internet gateway

Reply:
Rsolved in the last commits, please, check.